index.md

```diff
@@ -1,12 +1,12 @@
 ---
 layout: default
-title: Hyperledger TOC
+title: LF Decentralized Trust TAC
 nav_order: 1
 has_children: true
 has_toc: true
 ---
 [//]: # (SPDX-License-Identifier: CC-BY-4.0)
 
-# The Hyperledger Technical Oversight Committee
+# The LF Decentralized Trust Technical Advisory Council
 
-The Hyperledger Technical Oversight Committee (TOC) is responsible for oversight of the Hyperledger Foundation technical communities, including approving new proposals for project lifecycle changes, establishing community norms, workflows, or policies that are not within the scope of any single project, and resolving technical matters that affect multiple projects.
+The Hyperledger Technical Advisory Council (TAC) is responsible for oversight of the Hyperledger Foundation technical communities, including approving new proposals for project lifecycle changes, establishing community norms, workflows, or policies that are not within the scope of any single project, and resolving technical matters that affect multiple projects.
```

Comment:
Replace: `Hyperledger` and `Hyperledger Foundation` with `LF Decentralized Trust`.